content/en/about/version-policy/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Versioning Policy
+description: Supported versions of various components and dependencies of Istio.
+weight: 15
+icon: cadence
+---
+
+## Istio Versioning
+
+Istio versions are expressed as **x**.**y**.**z**, where **x** is the major version, **y** is the minor version, and **z** is the patch version.
+
+Istio provides support for each minor release for 3 months after the next minor release. For example, Istio 1.0 support will be dropped 3 months after Istio 1.2 is released. See [Build & Release Cadence](/about/release-cadence/) for more details.
+
+## Component Version Skew
+
+Various components of Istio that interact with each other must be kept within supported version ranges. Below describes the required version skews. Where possible, we always recommend using the same version between all components.
+
+### Istioctl
+
+We currently recommend using `istioctl` within the same minor version as both the control plane and data plane.
+
+For example, `istioctl` version **1.0.1** could be used with control plane version **1.0.0** and data plane version **1.0.6**.
+
+### Control/Data Plane
+
+The data plane (both sidecars and gateways) must be either the same version as the control plane, or at most 1 minor version older.
+
+For example, the data plane version **1.0.0** could be used with control plane version **1.0.0** or **1.1.0**.
+
+This implies that the control plane must be upgraded first. For more information, see the [Upgrade](/docs/setup/upgrade/) documentation.
+
+## Kubernetes versions
+
+The following table indicates the versions of Kubernetes that are supported for various Istio versions. While Istio may work on versions outside of those listed below, we do not test the stability or safety of these versions. Sticking within the supported version range is highly recommended.
+
+|Istio Version       | Kubernetes Version
```

Comment:
This table seems out of place in this doc. Everything else is just generic rules, but this table is listing release specific information. Do we even need such a historical summary? Even if we do, I'm not sure this doc is a good place for it.

Reply:
wrong place: @frankbu @howardjohn and I were working on the same topic and we weren't aware of each other's work. I have started an RFC to obtain some TOC agreement on our objectives. See: https://docs.google.com/document/d/14Xfe9V0vv8d7W75c21d-cY2XyK6xvRH0_Kg8j_zfh4Y/edit#. John and I were likely thinking alike that a table helps (us) map the Istio releases to Ks8 releases, so we can develop a holistic policy. :)

Reply:
If we do want a table like this, I think it should appear with the releases, not in this general policy doc. How about putting the table on this page: https://istio.io/news/releases/. You can also link to it from this doc.

Reply:
The release landing page seems like an odd location for this to me. I don't think many go there? I can put it on the install page

Reply:
My thought is that it's a summary of the k8s versions that each release note says it supports, so that's really where the table belongs, and it keeps release specific info out of the mainline docs.

I agree that people don't go to the release landing page often, but that doesn't matter if we link it from the places where users will go, like from this page and possibly other places where it might make sense.